Xamarin.Forms.Platform.Android/Renderers/SearchBarRenderer.cs

```diff
@@ -60,7 +61,8 @@ public override SizeRequest GetDesiredSize(int widthConstraint, int heightConstr
 
 		protected override SearchView CreateNativeControl()
 		{
-			return new SearchView(Context);
+			var context = (Context as ContextThemeWrapper).BaseContext ?? Context;
```

Comment:
- If this is only broken on APIi 19 then shouldn't we only do this if it's running on an API 19 device?
- I'm still curious if this also breaks if you have a Material Frame with a SearchView. If it doesn't than I wonder if the styles are setup wrong for the scrollview?
- if we end up going this route then I think we'd want to use this extension method . https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/ContextExtensions.cs#L101

Reply:
- I think we could have a check
- It doesn't show the issue if i use material frame with a scrollview.
- It fixes the issue (crash) if i use the ThemeContext, but applies the theme to the SearchBar, for example on the gallery the SearchBar will now have a white text color i think that will be a breaking change